apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -79,7 +80,23 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
   const locationFormSchema = z.object({
     locationType: z.string(),
     locationAddress: z.string().optional(),
-    locationLink: z.string().url().optional(),
+    locationLink:
+      selection?.value === LocationType.Whereby
+        ? z
+            .string()
+            .regex(/^(https:\/\/)?(www.)?whereby.com\/[a-zA-Z0-9]*/)
+            .optional()
+        : selection?.value === LocationType.Around
+        ? z
+            .string()
+            .regex(/^(https:\/\/)?(www.)?around.co\/[a-zA-Z0-9]*/)
+            .optional()
+        : selection?.value === LocationType.Riverside
+        ? z
+            .string()
+            .regex(/^(https:\/\/)?(www.)?riverside.fm\/studio\/[a-zA-Z0-9]*/)
+            .optional()
+        : z.string().url().optional(),
```

Comment:
hmmm i am not a big fan of hardcoding apps into the core product. @zomars any idea how we can abstract this into the app-store folder?